### PR TITLE
Storybook router

### PR DESCRIPTION
**Pull request by jmripper**
added in the storybook router

### CODE REVIEW

File: client/src/components/NavMenu/NavMenu.css [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+        flex-flow: column;
+        justify-content: flex-start;
+        align-content: space-around;
+        align-items: center;
+        height: 100vh;
+        width: 100vw;
+        margin: 0 auto;
+}
+
+.nav-container nav {
```
Comment (mrrogercampbell) [inline client/src/components/NavMenu/NavMenu.css]: Could you use a specific class name here?

File: client/src/components/NavMenu/NavMenu.css [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+        text-align: center;
+        padding-top: 4rem;
+}
+
+.nav-container nav ul {
```
Comment (mrrogercampbell) [inline client/src/components/NavMenu/NavMenu.css]: Could you use a specific class name here?

File: client/src/components/NavMenu/NavMenu.js [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+import './NavMenu.css'
+
+const NavMenu = (props) => (
+    props.isVisible ?
+        <div className="nav-container">
+            <nav>
+                <ul>
+                    <li>
+                        <Link to="#">
+                            <h3 className="nav-title">Home</h3>
```
Comment (mrrogercampbell) [inline client/src/components/NavMenu/NavMenu.js]: Please fix your indentations here. Might be helpful for you to install a React linter.

**Review by mrrogercampbell — COMMENTED** [review]
Over all this looks really good. Drop in a view comments below, please review. Let me know if you have any questions.

File: client/src/components/NavMenu/NavMenu.js [hunk 4]
```diff
@@ -0,0 +1,46 @@
+import React, { Component } from 'react';
```
Comment (mrrogercampbell) [inline client/src/components/NavMenu/NavMenu.js]: You do not need to import `Component` if you are not going to use it.

**Review by mrrogercampbell — COMMENTED** [review]
Also when I try to run your branch it is throwing errors. Does this code compile for you?

**Comment by mrrogercampbell** [comment]
Also please add a link to the issue for this feature. If there is not one create please make on. Thanks!

**Review by 0xZakk — CHANGES_REQUESTED** [review]
@jmripper There's a lot of stuff in here from your work on the nav menu. Can you fix this so it's just the storybook react-router stuff?

**Comment by jmripper** [comment]
@ZakkMan does that mean to delete all the nav menu stuff I have? I just want to make sure that's what you're saying.

**Comment by 0xZakk** [comment]
@jmripper Know we spoke about this in person, but so we have it all here too: you don't want to delete the files, you just want to remove them from this PR

**Review by 0xZakk — APPROVED** [review]

**Comment by 0xZakk** [comment]
@jmripper Fantastic job!

**Pull request merged**